salt/modules/monit.py

```diff
@@ -149,3 +151,105 @@ def status(svc_name=''):
     else:
         ret = entries.get(svc_name, 'No such service')
     return ret
+
+
+def reload():
+    '''
+    .. versionadded:: Boron
+
+    Reload monit configuration
+
+    CLI Example:
+
+    .. code-block:: bash
+
+        salt '*' monit.reload
+    '''
+    cmd = 'monit reload'
+    return not __salt__['cmd.retcode'](cmd, python_shell=False)
+
+
+def configtest():
+    '''
+    .. versionadded:: Boron
+
+    Test monit configuration syntax
+
+    CLI Example:
+
+    .. code-block:: bash
+
+        salt '*' monit.configtest
+    '''
+    ret = {}
+    cmd = 'monit -t'
+    out = __salt__['cmd.run_all'](cmd)
+
+    if out['retcode'] != 0:
+        ret['comment'] = 'Syntax Error'
+        ret['stderr'] = out['stderr']
+        ret['result'] = False
+        return ret
+
+    ret['comment'] = 'Syntax OK'
+    ret['stdout'] = out['stdout']
+    ret['result'] = True
+    return ret
+
+
+def version():
+    '''
+    .. versionadded:: Boron
+
+    Return version from monit -V
+
+    CLI Example:
+
+    .. code-block:: bash
+
+        salt '*' monit.version
+    '''
+    cmd = 'monit -V'
+    out = __salt__['cmd.run'](cmd).splitlines()
+    ret = out[0].split()
+    return ret[-1]
+
+
+def id(**kwargs):
```

Comment:
@cachedout Sure, I will do the update for the code and the doc string, you suggest to have ```def id(reset=False)``` instead ?